docs/tutorial.md

```diff
@@ -0,0 +1,190 @@
+
+### Setup environment
+- Clone
+
+```sh
+git clone https://github.com/mobvoi/wenet.git
+```
+
+
+
+- Install Conda
+
+https://docs.conda.io/en/latest/miniconda.html
+
+
+- Create conda env
+
+pytorch 1.6.0 is suggested. We meet some error on NCCL when using 1.7.0 on 2080 Ti.
+
+```
+conda create -n wenet python=3.8
+conda activate wenet
+pip install -r requirements.txt
+conda install pytorch==1.6.0 cudatoolkit=10.1 -c pytorch
+```
+
+- Install Kaldi
+
+Wenet need Kaldi to extract feature (a torchaudio version is in developing)
+
+Download and build [Kaldi](https://github.com/kaldi-asr/kaldi).
+
+Set the Kaldi root
+```
+vim example/aishell/s0/path.sh
+KALDI_ROOT=your_kaldi_root_path
+```
+
+
+### First Experiment
+
+We provide a recipe example/aishell/s0/run.sh on aishell-1 data.
+
+The recipe is simple and we suggest you run each stage one by one manually and check the result to understand the whole process.
+
+```
+cd example/aishell/s0
+bash run.sh --stage -1 --stop-stage -1
+bash run.sh --stage 0 --stop-stage 0
+bash run.sh --stage 1 --stop-stage 1
+bash run.sh --stage 2 --stop-stage 2
+bash run.sh --stage 3 --stop-stage 3
+bash run.sh --stage 4 --stop-stage 5
+bash run.sh --stage 5 --stop-stage 5
+```
+
+You could also just run the whole script
+```
+bash run.sh --stage -1 --stop-stage 5
+```
+
+
+
+#### stage -1 Download data
+
+This stage downloads the data to the local path `$data`. This may take several hours.  If you already downloaded the aishell-1 data. please change the `$data` variable in `run.sh` and start from `--stage 0`.
+
+#### stage 0 Prepare Kaldi format
+
+In this stage, `local/aishell_data_prep.sh` organizes the original aishell-1 data into a Kaldi format. In the kaldi dataset format, each dataset includes the two most important files `wav.scp`, `text`, and other resource files.
+
+In this demo recipe, this stage also uses Kaldi script `utils/perturb_data_dir_speed.sh` to do speed perturb augmentation on the training data. With 0.9 and 1.1 speeding up, the size of the training data is 3 times of the original data.
+
+please see `data/train/wav.scp` and `data/train/text`. 
+If you want to train your customized data, just organize the data into two files wav.scp and text, and start from `stage 1`.
+c
+
+
+#### stage 1 Extract acoustic feature
+
+In this stage, Kaldi script `steps/make_fbank_pitch.sh` is used to extract the acoustic feature from the wavs for train, test, and dev set.
+
+`compute-cmvn-stats` is used to extract global cmvn statistics. These statistics will be used to normalize the acoustic feature. Set `cmvn=false` will ignore this step.
+
+See the generated training feature file in `fbank_pitch/train/feat.scp`.
+
+#### stage 2 Generate label token dictionary
+
+The dict, a map between label tokens(we use the character for Aishell-1) and
+ the integer indexes.
```

Comment:
<blank> denotes blank symbol for CTC.
<unk> denotes unknown token, any out of vocabulary token will be mapped into <unk>.
<sos/eos> denotes sos and eos symbol for attention encoder decoder training, and they shares the same id.